frontend/src/features/properties/map/research/ResearchContainer.tsx

```diff
@@ -39,18 +39,19 @@ export const ResearchContainer: React.FunctionComponent<IResearchContainerProps>
   );
 
   const [showConfirmModal, setShowConfirmModal] = useState<boolean>(false);
+  const { search } = useMapSearch();
 
   const menuItems = researchFile?.researchProperties?.map(x => getPropertyName(x)) || [];
   menuItems.unshift('RFile Summary');
 
-  useEffect(() => {
-    async function fetchResearchFile() {
-      var retrieved = await retrieveResearchFile(props.researchFileId);
-      setResearchFile(retrieved);
-      setIsDirty(false);
```

Comment:
avoid using flags to trigger functions where possible. In this case the isDirty flag was causing the research file to be retrieved an additional time during an update.